### PR TITLE
Features/total issuance

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/modules/peers/peer.service.ts [hunk 1]
```diff
@@ -84,18 +84,35 @@ export class PeerService {
...
     }
     const networkParam = this.networkParams.find((item) => item.type === network);
     const treasuryAccount = stringToU8a('modlpy/trsry'.padEnd(32, '\0'));
-    const {data: {free: balance}} = await networkParam.api.query.system.account(treasuryAccount);
-    const formatedBalance = +balance.toString() / 10 ** 15;
-    return formatedBalance
+    const {
+      data: {free: balance},
+    } = await networkParam.api.query.system.account(treasuryAccount);
+    const formatedBalance = formatBalance(balance, {decimals: 15});
```
Comment (evgeny-s) [inline src/modules/peers/peer.service.ts:100]: @karthikeyan-cere , could you please move all the mentions of decimals (15) to the single place and put it to the ENV file? 
We can easily change it when we applied a new chainSpec with modified decimals.

Comment (evgeny-s) [reply src/modules/peers/peer.service.ts:100]: Check all other places, not only this file

**Review by evgeny-s — CHANGES_REQUESTED** [review]
Check comments

File: src/modules/peers/peer.service.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/modules/peers/peer.service.ts:100]: @evgeny-s, This is a temporary solution only, Once we update decimals in our network via chainSpec, we will remove the option from formatBalance().

**Review by evgeny-s — APPROVED** [review]

**Pull request merged**